### ISSUE TITLE
Compatibility with networkx v.2.0

### ISSUE THREAD

**Issue by kavvkon** [Pyomo/pyomo#261]
networkx version 2.0 is backwards incompatible

``` python
  File "c:\program files (x86)\anaconda2\lib\site-packages\pyomo\pysp\scenariotree\instance_factory.py", line 396, in _init
    self._model_module.pysp_scenario_tree_model_callback()
  File "C:/Users/kavvkon/AppData/Local/Local Documents - no backup/git/a/a\ph_model.py", line 90, in pysp_scenario_tree_model_callback
    stage_names=stagenames]
  File "c:\program files (x86)\anaconda2\lib\site-packages\pyomo\pysp\scenariotree\tree_structure_model.py", line 211, in ScenarioTreeModelFromNetworkX
    root = [u for u,d in tree.in_degree().items() if d == 0]
AttributeError: 'InDegreeView' object has no attribute 'items'

```

**Comment by ghackebeil** [comment]
I believe this is fixed on the pysp development branch (pysp_solver_interface), which will be merged into master very soon.

**Comment by ghackebeil** [comment]
Fixed on master as of the merge in 101bbc3cddee421d895238de86cdbe5d3fef06be
